lib/src/stack.dart

```diff
@@ -86,10 +86,10 @@ class CircularChartStackTween extends Tween<CircularChartStack> {
 
   @override
   CircularChartStack lerp(double t) => new CircularChartStack(
-        begin.rank,
-        lerpDouble(begin.radius, end.radius, t),
-        lerpDouble(begin.width, end.width, t),
-        lerpDouble(begin.startAngle, end.startAngle, t),
+        begin!.rank,
+        lerpDouble(begin!.radius, end!.radius, t) ?? 0.0,
+        lerpDouble(begin!.width, end!.width, t) ?? 0.0,
+        lerpDouble(begin!.startAngle, end!.startAngle, t) ?? 0.0,
```

Comment:
`begin` is required in the `CircularChartStackTween` initializer, but optional in the superclass, so it's safe to force-unwrap here (see line 78)

`lerpDouble` has a fail-able initializer so we're defaulting to 0.0 in that case